sinch/domains/verification/models/responses.py

```diff
@@ -12,22 +31,22 @@ class StartVerificationResponse(SinchBaseModel):
 
 @dataclass
 class StartSMSInitiateVerificationResponse(StartVerificationResponse):
-    sms: dict
+    sms: SMSResponse
 
 
 @dataclass
 class StartFlashCallInitiateVerificationResponse(StartVerificationResponse):
-    flashcall: dict
+    flashcall: FlashCallResponse
```

Comment:
The server returns a property named `flashCall`. Is there a reason to change it to `flashcall`?

Reply:
I want to be consistent and return values in `snake_case` not `camelCase`.
`flash_call` vs `flashcall` ? What's better? Wdyt?

Reply:
That's up to you 🙂